x-pack/plugins/security_solution/public/timelines/components/open_timeline/open_timeline.tsx

```diff
@@ -30,6 +31,11 @@ import * as i18n from './translations';
 import { OPEN_TIMELINE_CLASS_NAME } from './helpers';
 import { OpenTimelineProps, OpenTimelineResult, ActionTimelineToShow } from './types';
 
+const SelectableQueryText = styled.span`
```

Comment:
nit: SelectableQueryText doesn't seem to match the scenario here, please consider rename it. Aside from this, all looks good, verified on Chrome, Safari and Firefox. Thanks for fixing this @machadoum!